app/models/merchant.rb

```diff
@@ -66,10 +66,10 @@ def self.filter_by_disabled
   end
 
   def disabled_items
-    items.where(enabled: "disabled")
+    items.where(enabled: 1)
```

Comment:
You're going to have to catch me up on this because enum functionality works fine on items. On main go to rails console and you'll have all of usual AR methods on items.
```
Item.first.enabled? > 'true'
Item.first.disabled? > 'false'
Item.first.enabled > 'disabled'
Item.where(enabled: 'disabled') > AR objects
```